gherkin/php/src/StringUtils.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Cucumber\Gherkin;
+
+/**
+ * Keeps common string operations in one place using correct unicode functions
+ * (and normalises naming with other implementations)
+ */
+final class StringUtils
+{
+    private const DEFAULT_ENCODING = 'utf-8';
+
+    // same patterns as the Java implementation
+    private const WHITESPACE_PATTERN = "[ \\t\\n\\x0B\\f\\r\\x85\\xA0]+";
+    private const WHITESPACE_PATTERN_NO_NEWLINE = "[ \\t\\x0B\\f\\r\\x85\\xA0]+";
+
+    public static function symbolCount(string $string): int
+    {
+        return mb_strlen($string, self::DEFAULT_ENCODING);
```

Comment:
to check if the mb polyfill is there or the extension required

Reply:
Yep it's in composer.json